infra/token-transfer-server/src/config.js

```diff
@@ -45,8 +45,7 @@ if (!sessionSecret) {
 const largeTransferThreshold =
   Number(process.env.LARGE_TRANSFER_THRESHOLD) || 100000
 
-const largeTransferDelayMinutes =
-  Number(process.env.LARGE_TRANSFER_DELAY_MINUTES) || 60
+const largeTransferDelayMinutes = process.env.LARGE_TRANSFER_DELAY_MINUTES || 60
```

Comment:
Why removing the cat to Number?

Reply:
If you set the environment variable to 0 it appears as a string here, then it does `Number("0") || 60` which is always 60 because it is doing 0 || 60. The value gets passed to moment.subtract which is happy with a number or a string anyway.

Reply:
sg. ty